src/LifeCycle.hs

```diff
@@ -70,8 +69,6 @@ thawPhase now = (phaseStatus     %~ thawStatus)
 phaseTrans :: Phase -> PhaseChange -> Maybe (Phase, [PhaseAction])
 phaseTrans (PhaseRefinement ActivePhase{}) RefinementPhaseTimeOut
     = Just (PhaseJury, [JuryPhasePrincipalEmail])
-phaseTrans (PhaseRefinement ActivePhase{}) RefinementPhaseMarkedByModerator
```

Comment:
thanks, that's a good point.  the above constructor didn't implement this requirement, though, hence removing it strictly improves the situation (less dead code, same feature coverage).

i've implemented this feature (in a slightly liberal interpretation, plus the analogous one for voting phase) in #483.  took 3'39'' with testing.  (-: